modules/price/src/Calculator.php

```diff
@@ -180,6 +180,32 @@ public static function trim($number) {
   }
 
   /**
+   * Rounds the number to specified precision.
+   *
+   * @param string $number
+   *   The number to trim.
+   * @param int $precision
+   *   The number of decimal digits to round to.
+   * @param int $mode
+   *   The rounding mode.
+   *
+   * @return string
+   *   The rounded number.
+   */
+  public static function round($number, $precision = 0, $mode = PHP_ROUND_HALF_UP) {
```

Comment:
I am also agree that the addition of price rounding is a good thing but it need to be implemented in more places of drupalcommerce just to see the effect.

The first useful place that I've found is in **modules/order/src/Entity/OrderItem.php** +245

    protected function recalculateTotalPrice() {
      if ($unit_price = $this->getUnitPrice()) {
        $this->total_price = $unit_price->multiply($this->getQuantity())->round();
      }
    }

Works great with default precision and rounding mode but it would be better to make these values configurable in **modules/price/src/Plugin/Field/FieldWidget/PriceDefaultWidget.php**